apps/system/utils/security_level_cmd.c

```diff
@@ -0,0 +1,131 @@
+/****************************************************************************
+ *
+ * Copyright 2023 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <tinyara/ss_slot_index.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/types.h>
+#include <sys/prctl.h>
+#include <apps/shell/tash.h>
+#include <security/security_ss.h>
+
+/********************************************************************************
+ * Private Functions
+ ********************************************************************************/
+static void show_usage(void) 
+{
+	printf("\nUsage : sec_level [OPTION]...\n");
+	printf("Set your security level high or low, or get the current security level status.\n\n");
+	printf("Options :\n");
+	printf("high		set security level 1(high)\n");
+	printf("		You cannot see some secure information.\n\n");
+	printf("low		set security level 0(low)\n");
+	printf("		You can see all secure information.\n\n");
+	printf("status		show current value of security level\n\n");
+}
+
+static void tash_security_level(int argc, char **args)
+{
+        security_handle hnd;
+	security_data ss_data = {NULL, 0};
+	char ss_path[7];
+	security_error ret;
+
+	if (argc < 2) {
+		printf("Use [--help] option if you want to know how to use.\n");
+		return;
+	}
+
+	/* Show usage */
+	if (!strncmp(args[1], "--help", 7)) {
+		show_usage();
+		return;
+	}
+
+	/* Init security handler */
+	ret = security_init(&hnd);
+        if (ret != SECURITY_OK) {
+		printf("[security_init] error ret : %d\n", ret);
```

Comment:
This function is operation of user input. So, if we meet some failure case, we should reply failure of command execution first. And then we should add error type for development.